src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,23 @@
 package racingcar;
 
+
+import racingcar.controller.MainController;
+import racingcar.domain.CarFactory;
+import racingcar.service.WinnerDeterminer;
+import racingcar.validator.Validator;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+        InputView inputView = new InputView();
+        OutputView outputView = new OutputView();
+        Validator validator = new Validator();
+        CarFactory carFactory = new CarFactory();
+        WinnerDeterminer winnerDeterminer = new WinnerDeterminer();
```

Comment:
여기서 new로 객체를 생성하는 것 보다, config 클래스를 만들어서 의존성 주입을 통해 생성하는것도 좋을 거 같아요~ 
의존성 주입을 관리하는 클래스르 따로 두면, 나중에 유지보수 할 때 좋을 거 같아요. 또, 프레임워크로 넘어갈때도 코드 변경이 거의 없이 가능할 거예요.

```suggestion
        AppConfig appConfig = new AppConfig();
        MainController controller = new Controller(appConfig.inputView(), appConfig.ouputView(), appConfig.validator(), appConfig.carFactory(), appConfig.winnerDetermiter());      
```

위 코드가, 파라미터로 의존성 주입이 너무 많다고 생각되면, 정적 팩토리 메서드를 검색해서 사용하시면 좋을 거 같아요

Reply:
그렇게 하면 깔끔하게 처리할 수 있군요~~ 감사합니ㅏㄷ!!

Reply:
저도 과제를 할때마다 의존성 주입을 위한 객체 생성을 어디서 해야 하나 고민이 많았는데요!. 정적 팩토리 메서드를 사용해보니 좋았던 것 같습니다!! ㅎㅎ